central/blob/datastore/search/searcher_impl.go

```diff
@@ -0,0 +1,53 @@
+package search
+
+import (
+	"context"
+
+	"github.com/stackrox/rox/central/blob/datastore/store"
+	v1 "github.com/stackrox/rox/generated/api/v1"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/sac/resources"
+	"github.com/stackrox/rox/pkg/search"
+	searchAuth "github.com/stackrox/rox/pkg/search/auth"
+	"github.com/stackrox/rox/pkg/search/scoped/postgres"
+)
+
+type searcherImpl struct {
+	storage           store.Store
+	formattedSearcher search.Searcher
+}
+
+func (s *searcherImpl) SearchIDs(ctx context.Context, q *v1.Query) ([]string, error) {
+	results, err := s.formattedSearcher.Search(ctx, q)
+	if err != nil || len(results) == 0 {
+		return nil, err
```

Comment:
if nothing is found that will return nil, nil.  Probably fine.  simply mentioning for consideration in case 
we want to return something else in that case like a wrapped error or something else.